### PR TITLE
feat: tooltip component

### PR DESCRIPTION
**Pull request by dan-ziv**
### Description of the Changes

* Add tooltip component implementation.
* Add tooltip component to rewind and volume components.
* Add tooltip strings.

### CheckLists

- [x] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [x] test are passing in local environment 
- [x] Travis tests are passing (or test results are not worse than on master branch :))
- [x] Docs have been updated

### CODE REVIEW

File: src/components/rewind/rewind.js [hunk 1]
```diff
@@ -4,7 +4,8 @@ import {h} from 'preact';
 import {Localizer, Text} from 'preact-i18n';
```
Comment (AnnaAleksandrowicz) [inline src/components/rewind/rewind.js]: @dan-ziv  - even though this isn't a doc, should I perhaps be reviewing the tooltips?

**Comment by OrenMe** [comment]
@dan-ziv please resolve conflicts.
Which components have tooltip integrated beside volume and rewind?

**Comment by dan-ziv** [comment]
@OrenMe done.
For now just those you mentioned defined in in Hillel's mockups.

**Comment by kaltura-hooks** [comment]
Hi @dan-ziv,
Thank you for contributing this pull request!
Please sign the Kaltura CLA so we can review and merge your contribution.
Learn more at http://bit.ly/KalturaContrib

**Comment by OrenMe** [comment]
done in #468

**Pull request closed without merging**